test.js

```diff
@@ -69,6 +71,18 @@ test('do not include `stderr` in errors when set to `inherit`', async t => {
 	t.notRegex(err.message, /stderr/);
 });
 
+test('pass `stdout` to a file descriptor', async t => {
+	const file = tempfile('.txt');
+	await m('fixtures/noop', ['foo bar'], {stdout: fs.openSync(file, 'w')});
+	t.is(fs.readFileSync(file, 'utf8'), 'foo bar\n');
+});
+
+test('pass `stderr` to a file descriptor', async t => {
+	const file = tempfile('.txt');
+	await m('fixtures/noop-err', ['foo bar'], {stderr: fs.openSync(file, 'w')});
+	t.is(fs.readFileSync(file, 'utf8'), 'foo bar\n');
```

Comment:
Yes exactly. Either the docs are wrong, or I did it wrong. But the docs lik to [Stream](https://nodejs.org/dist/latest-v6.x/docs/api/stream.html#stream_stream) which states

<img width="1056" alt="screen shot 2017-03-23 at 19 32 53" src="https://cloud.githubusercontent.com/assets/1913805/24264086/88d6225e-0fff-11e7-91c7-d82034f53fe9.png">

> Incorrect value for stdio stream: WriteStream

But but... Ugh, nevermind.